sample/F8/sessions

```diff
@@ -0,0 +1,1308 @@
+[
+   {
+      "id":1,
+      "short_abstract":"",
+      "title":"Registration",
+      "subtitle":"day-1-registration",
+      "microlocation":{
+         "id":1,
+         "name":"Registration Desk"
+      },
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":1,
+         "name":"1 hour session",
+         "length":"01.00"
+      },
+      "track":{},
+      "start_time":"2017-04-18T18:00:00",
+      "end_time":"2017-04-18T19:00:00"
+   },
+   {
+      "short_abstract":"",
+      "id":2,
+      "title":"Breakfast",
+      "subtitle":"day-1-breakfast",
+      "microlocation":{
+         "id":2,
+         "name":"LL"
+      },
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
```

Comment:
timezone in event.json is "US/Mountain" while over here its UTC. please correct it.